src/actions/databases.js

```diff
@@ -1,4 +1,4 @@
-const getDB = require('remote').require('./src/db').getDB;
+import { getDBSession } from '../db';
```

Comment:
I think the concept of a `Server Session` fits to the requirements due to the fact that the *back-end* of the application must handle tunnels and connections issues like a drop of a connection.